pom.xml

```diff
@@ -102,15 +106,16 @@
     <scmTag>HEAD</scmTag>
     <!-- Where to put temporary files during test runs. -->
     <surefireTempDir>${project.build.directory}/tmp</surefireTempDir>
+    <spotbugs-maven-plugin.version>3.1.10</spotbugs-maven-plugin.version>
   </properties>
 
   <dependencyManagement>
     <dependencies>
       <!-- static analysis -->
       <dependency>
-        <groupId>com.google.code.findbugs</groupId>
-        <artifactId>annotations</artifactId>
-        <version>3.0.0</version>
+        <groupId>com.github.spotbugs</groupId>
+        <artifactId>spotbugs-annotations</artifactId>
+        <version>${spotbugs-maven-plugin.version}</version>
```

Comment:
Not sure it will be always correct. https://spotbugs.readthedocs.io/en/stable/migration.html already references 3.1.11, and I am not sure the versions will be aligned forever. I would recommend removing the property and to version things separately, but t is not a strong opinion

Reply:
Sure thing :)